src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/GerritMessageProvider.java

```diff
@@ -49,7 +49,7 @@
      * @param build Triggered build to provide custom message for
      * @return the custom message
      */
-    public String getBuildStartedMessage(AbstractBuild build) {
+    public String getBuildStartedMessage(Run build) {
```

Comment:
I hadn't seen that one, but I know there are some issues lying ahead for some other external sony lib that assumes `AbstractBuild` or `AbstractProject`. I forget the name of it right now but will tell you when I hit it again.